internal/pkg/runtime/launcher/oci/launcher_linux.go

```diff
@@ -90,6 +90,13 @@ func NewLauncher(opts ...launcher.Option) (*Launcher, error) {
 		return nil, err
 	}
 
+	// We are emulating native mode `--compat`, so  we provide a user-writable
+	// tmpfs by default, unless `--no-compat` was requested without
+	// `--writable-tmpfs`.
+	if !lo.NoCompat || lo.WritableTmpfs {
```

Comment:
Coding style question: this line only works by virtue of operator precedence rules. Of course, there are certain cases where we wouldn't consider that a barrier to readability (`x+y*z`). My question is whether the precedence of unary negation over binary disjunction is also in that class, or whether wrapping `!lo.NoCompat` in parentheses would be preferable in such cases.

Reply:
Generally you wouldn't wrap with braces for a `!`. Note that `gofmt` puts the `!` right against the `lo.NoCompat`, but spaces around the `||` to clarify precedence.

FWIW - I'd be more inclined to use parentheses for (x+y*z)